Vegeting/Screens/CreateGroup/ViewController/FirstCreateGroupViewController.swift

```diff
@@ -90,22 +99,25 @@ final class FirstCreateGroupViewController: UIViewController {
         label.text = "오늘부터 한달 이내로만 모임을 만들 수 있어요."
         label.font = .preferredFont(forTextStyle: .subheadline)
         label.textColor = .gray
-        label.isHidden = true
         return label
     }()
     
     private let numberOfGroupPeopleTitleLabel: UILabel = {
         let label = UILabel()
         label.text = "모임 인원을 선택해주세요."
         label.font = .preferredFont(forTextStyle: .title2, compatibleWith: .init(legibilityWeight: .bold))
-        label.isHidden = true
         return label
     }()
     
     private lazy var numberOfGroupCollectionView: NumberOfGroupPeopleView = {
-        let view = NumberOfGroupPeopleView()
+        let view: NumberOfGroupPeopleView
+        switch entryPoint {
+        case .create:
+            view = NumberOfGroupPeopleView()
+        case .revise:
+            view = NumberOfGroupPeopleView(selectedNumber: club?.maxNumberOfPeople)
+        }
         view.delegate = self
```

Comment:
p2;
위에 언급한대로 이것도 생성자대신 이러한 형태라면 가독성이나, 유지보수에 더 용이할 거 같습니다~!
```suggestion
let numberOfGroupPeopleView = NumberOfGroupPeopleView()
        if entryPoint == .revise, let maximum = club?.maxNumberOfPeople {
            groupCategoryView.selectAlreadyCategory(selectedCategory: maximum)
        }
        numberOfGroupPeopleView.delegate = self
        return numberOfGroupPeopleView
```

Reply:
[ee60d72](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/164/commits/ee60d72c8868fc88a065c29f3cd72b9c42ffb49e), [6498754](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/164/commits/6498754041f6b0398a18aaf6bd0ec4278a1db033)에서 반영 완료

Reply:
p4;
두가지 모두 select와 관련된 함수명으로 바꿔주시면 더욱 좋을거 같습니다~~